File: app/src/main/java/netpro/keyTransmitter/LongPressKey.java
package netpro.keyTransmitter;

import android.util.Log;

public class LongPressKey extends Key {

    private static final long serialVersionUID = -7412635502613056996L;

    public LongPressKey(int columnSpan, int rowSpan, String name, String description, Type type) {
        super(columnSpan, rowSpan, name, description, type);
    }

    @Override
    public void onLongClick() {
        Log.d("KeyViewHolder", "長押し");
    }
}

File: app/src/main/java/netpro/keytransmitter/LongPressKey.kt
package netpro.keytransmitter

/**
 * 長押ししたときにキーを送信するボタンです
 */
class LongPressKey(columnSpan: Int, rowSpan: Int, description: String, type: KeyType) : BaseKey(columnSpan, rowSpan, description, type) {

    val NORMAL = "normal"

    override fun onLongClick() {
        send(keyCodesMap[NORMAL])
    }

    companion object {
        private val serialVersionUID = -7412635502613056996L
    }
}
